istio-cni/values_gke.yaml

```diff
@@ -1,18 +1,19 @@
-hub: docker.io/tiswanso
-tag: v0.1-dev
-pullPolicy: Always
+cni:
+  hub: docker.io/tiswanso
```

Comment:
not related to this change but this doesn't seem right

Reply:
Should i need to address this in separate PR?

Reply:
we could use docker.io/istiocni or gcr.io like:
```
cni:
  hub: gcr.io/istio-release
  tag:  master-latest-daily
```